schematic/utils/df_utils.py

```diff
@@ -40,7 +42,12 @@ def load_df(file_path, preserve_raw_input=True, data_model=False, **load_args):
         #Cast the columns in dataframe to string while preserving NaN
         null_cells = org_df.isnull() 
         org_df = org_df.astype(str).mask(null_cells, '')
-        ints = org_df.applymap(lambda x: np.int64(x) if str.isdigit(x) else False, na_action='ignore').fillna(False)
+
+        if org_df.size < 1000:
```

Comment:
@linglp @mialy-defelice I'll update that PR and merge it first. I'll add comments, but don't think we should make the "large manifest" size cutoff configurable, since below a certain size performance is actually worse with parallelization, and I wouldn't expect users to know what the cutoff would be and don't want to have them to have to use trial and error to find something that works.

Reply:
@GiaJordan I misspoke, I meant add it as a variable not an arg. When numbers are hard coded like this I think it makes it more clear what's happening.

Reply:
Ahh I see, yeah that would be better!